<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>进程</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>711</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>540</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>结束进程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="poweroffButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restartButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>540</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>重启</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>性能</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>591</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_cpu">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>16</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QCustomPlot" name="cpuCustomPlot" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>551</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="cpuName">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>481</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>利用率</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuUse">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>380</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>340</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>正在运行进程：</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuProcRun">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>340</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>440</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>正常运行时间</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuTime">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>480</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>400</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>已创建进程：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>460</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>被挂起进程：</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuProcCreate">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>400</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuProcBlock">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>460</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_mem">
       <attribute name="title">
        <string>Mem</string>
       </attribute>
       <widget class="QCustomPlot" name="memCustomPlot" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>551</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>内存</string>
        </property>
       </widget>
       <widget class="QLabel" name="memAll">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>使用量</string>
        </property>
       </widget>
       <widget class="QLabel" name="memUse">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>50</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0G</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="memProgressBar">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>521</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>400</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>使用中</string>
        </property>
       </widget>
       <widget class="QLabel" name="memUsing">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>440</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0G</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>400</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>可用</string>
        </property>
       </widget>
       <widget class="QLabel" name="memAvailable">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>440</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0G</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_disk">
       <attribute name="title">
        <string>Disk</string>
       </attribute>
       <widget class="QCustomPlot" name="diskCustomPlot" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>551</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>磁盘</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_net">
       <attribute name="title">
        <string>Net</string>
       </attribute>
       <widget class="QCustomPlot" name="netCustomPlot" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>551</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>网络</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>350</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QLabel" name="netSend">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>370</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0Kbps</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>450</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>接受</string>
        </property>
       </widget>
       <widget class="QLabel" name="netRecv">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>470</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0Kbps</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>360</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>适配器名称：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>430</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>连接类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>500</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IPv4 地址：</string>
        </property>
       </widget>
       <widget class="QLabel" name="netDev">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>360</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="netType">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>430</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="netIP">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>500</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
